one_fm/api/doc_methods/shift_request.py

```diff
@@ -37,9 +41,24 @@ def on_update(doc, event):
         workflow_approve_reject(doc, [get_employee_user_id(doc.employee)])
 
     if doc.workflow_state == 'Draft':
+        assign_approver(doc, doc.approver)
         send_workflow_action_email(doc,[doc.approver])
         validate_shift_overlap(doc)
 
+def assign_approver(doc, approver_id):

```

Comment:
If we are using assignment rule for assigning the shift request why we added the code for the same here